scripts/test/lint.py

```diff
@@ -369,7 +370,7 @@ def check(self, path):
 INSTALL_LINTS = (
     MissesImportCommonVm(),
     FirstLineDoesNotSetErrorAction(),
-    UsesInvalidCategory(),
+    #UsesInvalidCategory(),
```

Comment:
instead of removing this linter you need to modify it to check for the line: `$category =VM-Get-Category()`. I am ok if you only add the function `VM-Get-Category` in this PR and in a separate PR you modify all packages and adapt the linter.